account_payment_order/models/account_payment.py

```diff
@@ -33,3 +34,11 @@ def _compute_payment_method_line_fields(self):
                     )
                 )
         return res
+
+    def _compute_available_partner_bank_ids(self):
+        order_pays = self.filtered(lambda p: p.payment_order_id and p.payment_line_ids)
```

Comment:
If this finally arrives, you should put a docstring in this override stating why is put.